tests/test_wire.py

```diff
@@ -66,6 +66,24 @@ def test_valid_names(self):
         self.assertTrue(self.is_valid_str(' '))
         self.assertTrue(self.is_valid_str('#$)(*&#@_+!#)('))
 
+    def is_name_setter_str(self, ns):
+        test = pyrtl.WireVector(1,'test')
+        test.name = ns
+        return test.name == ns
+    
+    def test_invalid_name_setter(self):
+        with self.assertRaisesRegex(pyrtl.PyrtlError, "WireVector names must be strings"):
```

Comment:
Matching the entire error message makes this test very sensitive to the message's exact wording. We should be able to make minor changes to error messages without breaking tests. Consider just checking for `PyrtlError`, or checking for a key word in the message.